packages/client/src/utils.ts

```diff
@@ -22,6 +23,14 @@ const IDTokenSchema = s.type({
   at_hash: s.optional(s.string()),
 });
 
+interface UriInfo {
```

Comment:
TBD: Better to rename to `CallbackUriParameters`?

Reply:
good point